src/service/app/charts/mining.ts

```diff
@@ -57,30 +56,27 @@ export const getStatisticDifficultyHashRate = (dispatch: AppDispatch) => {
     })
 }
 
-const sliceStatistics = (data: Array<State.StatisticDifficultyUncleRateEpoch>) =>
-  data.slice(isMobile() ? Math.ceil(data.length / 2) : 0)
-
 export const getStatisticDifficultyUncleRateEpoch = (dispatch: AppDispatch) => {
   const data = fetchEpochChartCache(
     ChartCachedKeys.DifficultyUncleRateEpoch,
   ) as State.StatisticDifficultyUncleRateEpoch[]
 
   if (data) {
-    dispatchDifficultyUncleRateEpoch(dispatch, sliceStatistics(data))
+    dispatchDifficultyUncleRateEpoch(dispatch, data)
     return
   }
   fetchStatisticDifficultyUncleRateEpoch()
     .then((response: Response.Response<Response.Wrapper<State.StatisticDifficultyUncleRateEpoch>[]> | null) => {
       if (!response) return
       const { data } = response
-      const difficultyUncleRateEpochs = data.slice(Math.ceil(data.length / 2)).map(wrapper => ({
+      const difficultyUncleRateEpochs = data.map(wrapper => ({
         epochNumber: wrapper.attributes.epochNumber,
         difficulty: wrapper.attributes.difficulty,
         uncleRate: new BigNumber(wrapper.attributes.uncleRate).toFixed(4),
         epochTime: wrapper.attributes.epochTime,
         epochLength: wrapper.attributes.epochLength,
       }))
-      const dispatchData = sliceStatistics(difficultyUncleRateEpochs)
+      const dispatchData = difficultyUncleRateEpochs
```

Comment:
Why not use `difficultyUncleRateEpochs` directly?

Reply:
Wow, I was blinded by the code. Will be updated soon

Reply:
Updated in [cce299d](https://github.com/nervosnetwork/ckb-explorer-frontend/pull/1014/commits/cce299dcd45e8481bf887f06ec59e88929f91e2d)